src/contracts/core/AllocationManager.sol

```diff
@@ -645,6 +645,71 @@ contract AllocationManager is
         return totalMagnitude;
     }
 
+    /**
+     * @notice Returns the pending allocations of an operator for a given strategy and operatorSets
```

Comment:
Might be worth noting that the assumption is that there is only MAX 1 pending allocation and deallocation. Thats why you only need to read `latestSnapshot` or use the last index in `_queuedDeallocationIndices`

Reply:
ah yes. I will add a comment. good call